### DISCUSSION TITLE
What is the use in sysconfig dir?

### DISCUSSION THREAD

**Discussion by hongkongkiwi** [finit-project/finit#194] [Q&A] [answered]
I've got a couple of quick questions, what is the point in specifying the conf.d directory when compiling finit?

When specifying an env file, in the examples it shows /etc/conf.d/<name> so, in this case we already have a full path, why would a conf.d directory need to be specified at all during compilation?

My second question is, how can I specify multiple directories during compilation? By default both /etc/default & /etc/conf.d is specified, but when I specify a dir, I can only specify one when using --with-sysconfig-dir=

**Comment by troglobit** [comment, marked as answer]
We use another directory for our builds, this is just a neutral example.

You can't.  The code does not support multiple sysconfig dirs at the moment.  Supporting multiple was planned but after initial investigation too complex to support and test.
